dht.go

```diff
@@ -105,13 +111,16 @@ func NewConfig() *Config {
 		MaxNodes:                500,
 		CleanupPeriod:           15 * time.Minute,
 		SaveRoutingTable:        true,
+		PassivMode:              false,
```

Comment:
This is redundant with the RateLimit, right?

(In English we would spell it Passive, I think.)